modules/javafx.graphics/src/main/java/javafx/scene/image/ImageView.java

```diff
@@ -826,13 +853,93 @@ public boolean isSettable(ImageView n) {
             }
         };
 
-         private static final List<CssMetaData<? extends Styleable, ?>> STYLEABLES;
-         static {
+        private static final CssMetaData<ImageView,Number> FIT_WIDTH =
+            new CssMetaData<ImageView,Number>("-fx-fit-width",
+                SizeConverter.getInstance(), 0.0) {
+
+                @Override
+                public boolean isSettable(ImageView n) {
```

Comment:
general question: should we define params in new code as final?

Reply:
FYI this code was C&P from `Node` thus it follows the same code semantics and structure as previously existing code. Changing `ImageView` would require updating all other classes that follow the same programming idioms.

Reply:
sure, that's why I said 'general question' ;) Do not make sense to do it here. Just something that came in my mind when I saw your PR.